src/types/types.ts

```diff
@@ -249,6 +249,28 @@ export interface EdgeMemo {
   memoName?: string
 }
 
+export interface EdgeTxAsset {
+  pluginId: string
+  tokenId: string
+  nativeAmount: string
+}
+
+export interface EdgeTxInfoSwap {
+  type: 'swap' | 'swapOrderPost' | 'swapOrderFill'
+  orderId?: string
+  direction: 'from' | 'to'
```

Comment:
I question the need for this. In our exchange plugins this denotes which side is
the guaranteed amount - the payment or receive amount. In this API, we already
define a nativeAmount for both sides in EdgeTxAsset. We can just make
nativeAmount optional to denote an unknown swap destAsset amount for some order
with slippage.